action.yml

```diff
@@ -16,6 +16,9 @@ inputs:
   label-operator:
     required: false
     description: The behavior of the labels filter, AND to match all labels, OR to match any label, NOT to exclude any listed label (default is OR)
+  fields:
+    required: false
+    description: A json object containing custom fields like product and client
```

Comment:
This should not be a JSON object, it should be a multiline input containing `field=value` pairs.